### PR TITLE
Homeless Hack

### PR DESCRIPTION
**Pull request by RoryMacDonald**

### CODE REVIEW

File: posts/homeless_hack_day.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Homeless Hackday London
+
+At Made Tech, we run an internal Hack Day or Learning Day every month. We take time off our paid work to focus on learning something new (like a new programming language) or to hack together some tools that solve a problem we're encountering.
```
Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: No need for parens around (like a new programming language)

File: posts/homeless_hack_day.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+I thought I'd share some of the thinking behind this and take you through what we created at the hack. Hopefully this will inspire some other organisations to follow suit and use hack days for good!
+
+#### Default to open
+We have always been an open company and publishing our [company handbook](https://github.com/madetech/handbook), [frequently blogging](https://www.madetech.com/blog), [mentoring new programmers](https://github.com/madetech/handbook/blob/master/roles/engineer.md#mentor) or [writing books](https://www.amazon.co.uk/Building-High-Performance-Agile-Teams/dp/1544972687) were steps towards being even more open. It felt at odds with our culture to run hack days that were closed events. Homeless Hack was a good opportunity to rectify this. 
+
+#### Be a socially responsible SME
+Historically we've never given much thought to things like Corporate Social Responsibility. We're an SME, we've got less than 30 people working for us and we like to think we run the business ethically and fairly.
```
Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: fewer than 30

File: posts/homeless_hack_day.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Be a socially responsible SME
+Historically we've never given much thought to things like Corporate Social Responsibility. We're an SME, we've got less than 30 people working for us and we like to think we run the business ethically and fairly. 
+
+It's always felt like the world of CSR didn't really apply to us. e.g. It's for the big enterprises, who do bad things and need to offset the bad things by doing some good things. This was a good opportunity to course correct some of our thinking around this and invest some time and money into something that would do some social good. 
+
+#### What are you doing on Saturday?
+We asked everyone in the company to attend HomelessHack, though had just 7 people attend. The event fell on a Friday and Saturday and many people couldn't give up a day of their weekend to attend. Interestingly, we had quite a few people who originally said they would attend, but as the event got closer, had to drop out.
```
Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: Maybe something like:

We invited everybody in the company to attend HomelessHack, though as the event neared, the number whittled down to 7.

File: posts/homeless_hack_day.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+> Use real pricing data to create a campaign to highlight the difference between LHA and real market rates
+
+We collectively agreed on what we wanted to try and showcase on Saturday and then broke the work down into a number of independent chunks. We collaborated mostly through verbal communication, but also had a Trello board and Slack room for bits too. 
+
+We must have spent about 5hrs on Friday afternoon / evening working on the problem before calling it a night at approx. 9pm.  
+
+## Saturday - We're Live
+
+We reconvened on Saturday morning at ~10am and continued working on the problem. We knew the showcase was going to be at 4pm, so didn't have a huge amount of time to get everything working, but managed to pull it all together approx. 1hr before the showcase, so we could prepare a short presentation:
```
Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: Unnecessary abbreviation in approx.?

File: posts/homeless_hack_day.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+We've also shared what we've produced with the team at [Centrepoint](https://centrepoint.org.uk) & [Crisis](https://www.crisis.org.uk), so they an take this forward and use it for their policy work. 
+
+## What's Next
+It looks like Homeless Hack is gaining momentum. I've seen Homeless Hack events organised in Leeds, Manchester and Oxford. It's great to see the good work being picked up by other cities. 
+
+As a company, we'll definitely be attending more public hack days. We're already talking about the next one, potentially an [NHS Hack Day](http://nhshackday.com/) and we're hoping to get a larger turnout from Made Tech next time! 
+
+Finally, a huge thank you to [Janet Hughes](https://twitter.com/janethughes), [James Cattell](https://twitter.com/jacattell), [Gia Marcus](https://twitter.com/la_gaia) & [Richard Barton](https://twitter.com/cioportfolio). They organised the London event and made it a fantastic success. These type of events wouldn't happen without their hard work, so top work to them all.
```
Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: Gaia Marcus

Comment (chrisblackburn) [inline posts/homeless_hack_day.md]: Lots of 'work' in the final sentence

File: posts/homeless_hack_day.md (hunk 1 above)
Comment (zuren) [inline posts/homeless_hack_day.md]: (, but a subordinate clause would be alright, )

**Comment by chrisblackburn** [comment]
By the time we got to work there was very little time to actually stitch something together, I felt. I think the decision to zero in on a single data source was key - there were ideas floating around about pulling from multiple sources for a while.

**Pull request merged**